cmd/oci-image-validate/main.go

```diff
@@ -145,6 +145,10 @@ func (v *validateCmd) Run(cmd *cobra.Command, args []string) {
 		exitcode = 1
 	}
 
+	if exitcode == 0 {
+		v.stdout.Println("validation succeeded")
```

Comment:
Preline has printed the OK message, this line seems redundant. Other changes of this PR seem good.